### ISSUE TITLE
Viseme issue when ''convert for VRchat''

### ISSUE THREAD

**Issue by SugarCrashhhh** [esperecyan/VRMConverterForVRChat#66]
I have an issue when converting for VRchat Fcl_MTH_U and O look fine, but then in the visemes vrc.v_ou and oh look exactly the same and they do not quite look like an ''oh'' nor ''ou'' 

Pictures of the issue:
![Unity_V9Tz2rTktv](https://github.com/esperecyan/VRMConverterForVRChat/assets/96464830/b7b55857-3e42-4425-8eab-35ade64e4c91)
![Unity_qVATJiD4W8](https://github.com/esperecyan/VRMConverterForVRChat/assets/96464830/2a271577-0ea8-4f3a-8de9-a83cf18ef922)

Pictures of Fcl_MTH_O and U:
![Unity_YNJ5yU3np7](https://github.com/esperecyan/VRMConverterForVRChat/assets/96464830/4a2cf473-1e12-402e-9530-36aa6837cf3e)
![Unity_vJ8HIdE1sX](https://github.com/esperecyan/VRMConverterForVRChat/assets/96464830/93cc45d8-2d1e-46af-8a70-b253c8ba35b6)

I am using unity 2022 ver installed by the VCC, and running latest version of VRMConverterForVRChat

Is there a way to fix this? 
Thanks!

**Comment by esperecyan** [comment]
To generate viseme for VRChat, this extension uses the blend value of Blender add-on “Cats Blender Plugin” and only A, I, and O are used.

https://store.vket.com/items/122
https://github.com/absolute-quantum/cats-blender-plugin/blob/0.13.3/tools/viseme.py

**Comment by SugarCrashhhh** [comment]
I understand now that it works like Cats Blender Plugin, but this isn't right is it? 

The vrc.v_ou and vrc.v_oh are the same, as you can see in the first two pictures
So when you speak in VRchat, making an ''o'' or ''u'' sound looks awkward because of this

Could it be an issue with the newer Cats plugin or VRoid studio models? 
I used CATS plugin in blender before and the visemes looked right, now any model that goes through cats or VRMConverter has this issue

Do you get the same results when using the VRMConverter or Cats plugin?

Thank you

**Comment by esperecyan** [comment]
Yes, the algorithm has not changed for a long time and neither has the Cats Blender Plugin for a long time.

Although the algorithms for oh and ou are different, they may look the same depending on the Viseme from which they are converted.
See below for the actual algorithm.
https://github.com/absolute-quantum/cats-blender-plugin/blob/0.19.0/tools/viseme.py#L149-L159

There is no ability to customize this algorithm in this extension, but you can add a callback function that does the post-processing after the conversion is complete.

**Comment by esperecyan** [comment]
There is no reply, so I close this.
